src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -233,3 +233,20 @@ def load_command_table(self, _):
             'rollback',
             'aks_mesh_upgrade_rollback',
             supports_no_wait=True)
+
+    # AKS approuting commands
+    with self.command_group(
+        'aks approuting', managed_clusters_sdk, client_factory=cf_managed_clusters
+    ) as g:
+        g.custom_command('enable', 'aks_approuting_enable')
+        g.custom_command('disable', 'aks_approuting_disable', confirmation=True)
+        g.custom_command('update', 'aks_approuting_update')
+
+    # AKS approuting dns-zone commands
+    with self.command_group(
+        'aks approuting zone', managed_clusters_sdk, client_factory=cf_managed_clusters
+    ) as g:
```

Comment:
```suggestion
    with self.command_group('aks approuting zone', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
```

Reply:
Another issue with the line break

Created issue [#435](https://github.com/Azure/azure-cli-dev-tools/issues/435)

Reply:
don't forget this

Reply:
Totally missed this was a suggestion, pulling it in!

Reply:
added